app/components/map-utility-box.js

```diff
@@ -51,7 +51,8 @@ export default Ember.Component.extend({
   stops(mode) {
     // return an array of objects, each with a display-ready range and color
     const config = choroplethConfigs.find(d => d.id === mode);
-    const stops = config.paintFill['fill-color'].stops;
+    const stops = config.stops;
+    const colors = config.colors;
     const { isPercent } = config;
```

Comment:
I think these lines should be this: 
`const { isPercent, stops, colors } = config;`

I'll change these